WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -492,14 +488,14 @@ function(task, colIdx) {
 	var width = (AjxEnv.isIE || AjxEnv.isSafari) ? "22" : "16";
 
 	// first row
-	htmlArr[idx++] = "<table width=100% class='TopRow'>";
```

Comment:
Mail List items are using mainly based on divs whereas list view items are combinations of div , nested table and tds.

Yes, we can override specific styles for the task app. 
But, will it be useful here, in terms of semantics or layout ?
As mail list items and task list items are quite different in terms of layout,  in terms of elements or the alignment, spacing and placing of  of icons and text are different for both of them.